packages/example/components/chains/scdo/rpc.ts

```
@@ -4,8 +4,10 @@ import Axios from 'axios';
 // https://scdo-project.gitbook.io/scdo-wiki/developer/api/rpc
 export class ScdoNodeClient {
   public readonly axios: AxiosInstance;
+  private readonly rpcUrl: string;
 
-  constructor() {
+  constructor(rpcUrl: string = 'https://mainnet.scdo.org:8137') {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)